keyboards/punk75/config.h

```diff
@@ -24,7 +24,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRODUCT_ID 0x0001
 #define DEVICE_VER 0x0001
 #define MANUFACTURER dsanchezseco
-#define PRODUCT punk75
+#define PRODUCT dsanchezseco punk75
+#define DESCRIPTION dsanchezseco punk75
```

Comment:
Description isn't used by QMK, and product shouldn't repeat manufacturer. 
https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs

```suggestion
#define PRODUCT punk75
```

Reply:
I changed to follow the VIAs documentation to give the keyboard that functionality. https://caniusevia.com/docs/configuring_qmk#change-product it says is something that would be used in the future tho. I can reverted back if needed.

Reply:
VIA could very easily obtain the value of `MANUFACTURER` as well as `PRODUCT`, even if Windows will not display it. So there is no point in combining the two if that is why they "highly recommend" doing so.